apps/tutorial/public/docs/8-form-data-controlled/2-checkbox/prose.md

```diff
@@ -0,0 +1,41 @@
+Just like the controlled input, the _controlled checkbox_ is roughly the same approach, but with a different property and event binding combination. 
+
+Instead of setting value, we'll set `checked`.
+```hbs
+<input type="checkbox" checked={{@value}} />
+```
+
+And for the event binding, we'll use the `change` event.
+```gjs
+class Demo extends Component {
+  handleChange = (event) => {
+    let value = Boolean(event.target.checked);
+    this.args.onChange(value);
+  }
+
+  <template>
+    <input 
+      type="checkbox" 
+      value={{@value}} 
+      {{on 'change' this.handleChange}} />
+  </template>
+}
+```
+
+Checkboxes have a default `value` of `on` (as a string), which may not be a desired value to pass to your `onChange` handler. In this example, we can convert the `checked` property to a boolean.
+
+<p class="call-to-play">
+  Change the checkbox within the <code>ControlledInput</code> component 
+  to <strong>be controlled</strong>.
+</p>
+
+
+### References
```

Comment:
I started adding references to know APIs here, because it'd be useful for folks to reference official documentation -- but maintaining this manually is not great.

what I think I want to do is scan the markdown for certain keywords and generate the references at the bottom.